coalib/parsing/filters/LanguageFilter.py

```diff
@@ -0,0 +1,27 @@
+def language_filter(bears, languages, log_printer):
+    local_bears, global_bears = bears
```

Comment:
do we want to separate local and global bears? I wouldn't do that, especially as in the next-gen core we won't make a distinction any more in narrower sense, as their difference is only the degree in parallelization. The current core has special code for both of them, but the next-gen core won't. The bears do specify themselves the degree of parallelization^^